### PR TITLE
Releasing coq-mathcomp-multinomials (1.4)

### PR DESCRIPTION
**Pull request by strub**

### CODE REVIEW

File: released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.4/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "coq-mathcomp-algebra"
+  "coq-mathcomp-bigenough" {>= "1.0.0" & < "1.1~"}
+  "coq-mathcomp-finmap"    {>= "1.3.4" & < "1.4~"}
+]
+tags: [
+  "keyword:multinomials"
+  "keyword:monoid algebra"
+  "category:Math/Algebra/Multinomials"
+  "category:Math/Algebra/Monoid algebra"
+  "date:2016"
```
Comment (clarus) [inline released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.4/opam]:
```suggestion
  "date:2019-11-28"
```

File: released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.4/opam [hunk 2]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+name: "coq-mathcomp-multinomials"
+maintainer: "pierre-yves@strub.nu"
+homepage: "https://github.com/math-comp/multinomials-ssr"
+bug-reports: "https://github.com/math-comp/multinomials-ssr/issues"
```
Comment (clarus) [inline released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.4/opam]:
```suggestion
bug-reports: "https://github.com/math-comp/multinomials/issues"
```

File: released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.4/opam [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+]
+tags: [
+  "keyword:multinomials"
+  "keyword:monoid algebra"
+  "category:Math/Algebra/Multinomials"
+  "category:Math/Algebra/Monoid algebra"
+  "date:2016"
+  "logpath:SsrMultinomials"
+]
+synopsis: "A Multivariate polynomial Library for the Mathematical Components Library"
```
Comment (clarus) [inline released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.4/opam]:
```suggestion
synopsis: "A Multivariate polynomial Library for the Mathematical Components Library"
url: {
  src: "https://github.com/math-comp/multinomials/archive/1.4.tar.gz"
  checksum: "sha512=b8c4f750063b8764275ea3b0ca12b2bbc16d1c52c5b10bd71ef6d6bb0791ecfb1115681c4379ffe286ab8c550c1965ba08cd2c3a1e5600d8b1a17dee627e2aae"
}
```

File: released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.4/opam [hunk 4]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+name: "coq-mathcomp-multinomials"
+maintainer: "pierre-yves@strub.nu"
+homepage: "https://github.com/math-comp/multinomials-ssr"
```
Comment (clarus) [inline released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.4/opam]:
```suggestion
homepage: "https://github.com/math-comp/multinomials"
```

**Comment by clarus** [comment]
Hum... the tests are currently doing a 503... 
image

**Comment by clarus** [comment]
I think you should force-push to restart the tests.

**Comment by strub** [comment]
Ok, between me & gitlab, this has been a bit long, but I think this is ok now :)

**Comment by clarus** [comment]
Sweet

**Pull request merged**